src/server.ts

```diff
@@ -16,14 +17,130 @@ const app = express();
 app.use(express.json()); //add JSON body parser to each following route handler
 app.use(cors()); //add CORS support to each following route handler
 
-app.get("/", async (_req, res) => {
-    res.json({ msg: "Hello! There's nothing interesting for GET /" });
+// get all boards
+app.get("/boards", async (_req, res) => {
+    try {
+        const boards = await queryAndLog(client, "select * from boards");
```

Comment:
The object returned from the query is not a "board" but a fancy object from node-postgres, this naming is a bit misleading if you don't know that, especially if we weren't using typescript